distrax/_src/distributions/gamma.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2021 DeepMind Technologies Limited. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Gamma distribution."""
+
+from typing import Tuple, Union
+
+import chex
+from distrax._src.distributions import distribution
+from distrax._src.utils import conversion
+import jax
+import jax.numpy as jnp
+from tensorflow_probability.substrates import jax as tfp
+
+
+tfd = tfp.distributions
+
+Array = chex.Array
+Numeric = chex.Numeric
+PRNGKey = chex.PRNGKey
+
+
+class Gamma(distribution.Distribution):
+  """Gamma distribution with parameters
+  `concentration` (aka "alpha") and `rate` (aka "beta").
+  """
+
+  equiv_tfp_cls = tfd.Gamma
+
+  def __init__(self, concentration: Numeric, rate: Numeric):
```

Comment:
yes so since there is already a concept of `shape` i.e. in the batches etc. I thought I use the other names so as not to confound the two things... If you prefer the wikipedia convention I am happy to change it...

Reply:
Ah, good point. Let's leave is as is.